python/plugins/processing/algs/qgis/postgis_utils.py

```diff
@@ -153,7 +153,7 @@ def __init__(self, host=None, port=None, dbname=None, user=None,
         try:
             self.con = psycopg2.connect(self.con_info())
         except psycopg2.OperationalError as e:
-            raise DbError(unicode(e))
+            raise DbError(str(e))
```

Comment:
Can't we change it to `u'test'.format(u'testé')` then? We are not able to always control the full error string (e.g. it might contain a table name with non-standard ascii characters), so I think it's better to be prepared to handle unicode than to coerce everything into basic ascii.